<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>BMTE - Publications</title>
  <meta name="description" content="BMTE Research group -- Publications.">
  <link rel="stylesheet" href="https://mchaib.github.io/BMTE-Website/css/main.css">
  <link rel="canonical" href="https://mchaib.github.io/BMTE-Website/research/research_results/">
<link rel="shortcut icon" type ="image/x-icon" href="https://mchaib.github.io/BMTE-Website/images/logopic/favicon.ico">
<link rel="shortcut icon" href="https://mchaib.github.io/favicon.png">


<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-26505344-1', 'auto');
  ga('send', 'pageview');

</script>


</head>


  <body>

    <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container-fluid">
	<div class="navbar-header">
	  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-1" aria-expanded="false">
		<span class="sr-only">Toggle navigation</span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span> 
	  </button>
		

    <a class="navbar-brand" href="https://mchaib.github.io/BMTE-Website/">
    	<img style="position:relative; top:-25px" src="/BMTE-Website/images/logopic/group_logo_medium.png" height="40" width="auto" class="d-inline-block align-left" alt=""> 
    <a class="navbar-brand" href="https://mchaib.github.io/BMTE-Website/">
			  MBTE Research Group
    </a>
  	</a>
	</div>

<div class="collapse navbar-collapse" id="navbar-collapse-1">
	  <ul class="nav navbar-nav navbar-right">
		<li><a href="https://mchaib.github.io/BMTE-Website/">Home</a></li>
		<li><a href="https://mchaib.github.io/BMTE-Website/team">Team</a></li>
		<li><a href="https://mchaib.github.io/BMTE-Website/allprojects">Ongoing Projects</a></li>
		<li><a href="https://mchaib.github.io/BMTE-Website/publications">Publications</a></li>
		<li><a href="https://mchaib.github.io/BMTE-Website/synbio">SynBio</a></li>
	  </ul>
</div>

</div>

 <div class="language-switcher">
  
  
    <a href="https://mchaib.github.io/BMTE-Website/es/research/research_results/">üá™üá∏ / ESP</a>
  
</div>


</div>


    <div class="container-fluid">
      <div class="row">
        <div id="grid2id" class="col-sm-12">
  <h1 id="research-publications">Research publications</h1>

<h2 id="here-are-our-main-themes">Here are our main themes</h2>

<div class="row">

  <div class="col-sm-6 clearfix">
    <div class="well">

      <p><img src="https://mchaib.github.io/BMTE-Website/images/publications/msystems.webp" class="img-responsive" width="50%" style="float: top" /></p>

      <h3><pubtit>Comparative Metagenomic Analysis of Biosynthetic Diversity across Sponge Microbiomes Highlights Metabolic Novelty, Conservation, and Diversification</pubtit></h3>
      <p>Marine sponges and their microbial symbiotic communities are rich sources of diverse natural products (NPs) that often display biological activity, yet little is known about the global distribution of NPs and the symbionts that produce them. Since the majority of sponge symbionts remain uncultured, it is a challenge to characterize their NP biosynthetic pathways, assess their prevalence within the holobiont, and measure the diversity of NP biosynthetic gene clusters (BGCs) across sponge taxa and environments. Here, we explore the microbial biosynthetic landscapes of three high-microbial-abundance (HMA) sponges from the Atlantic Ocean and the Mediterranean Sea. This data set reveals striking novelty, with &lt;1% of the recovered gene cluster families (GCFs) showing similarity to any characterized BGC. When zooming in on the microbial communities of each sponge, we observed higher variability of specialized metabolic and taxonomic profiles between sponge species than within species. Nonetheless, we identified conservation of GCFs, with 20% of sponge GCFs being shared between at least two sponge species and a GCF core comprised of 6% of GCFs shared across all species. Within this functional core, we identified a set of widespread and diverse GCFs encoding nonribosomal peptide synthetases that are potentially involved in the production of diversified ether lipids, as well as GCFs putatively encoding the production of highly modified proteusins. The present work contributes to the small, yet growing body of data characterizing NP landscapes of marine sponge symbionts and to the cryptic biosynthetic potential contained in this environmental niche.</p>
      <p>Team members: <em>Catarina Loureiro, Anastasia Galani, Asimenia Gavriilidou, etc.</em></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
      <p><a data-toggle="collapse" href="#sponge-bib" class="btn-bib" style="text-decoration:none; color:#ebebeb; hover:#ebebeb;" role="button" aria-expanded="false">Selected papers</a></p>
      <div class="collapse" id="sponge-bib">
        <div class="well-abs">
          <div class="publications">
            <ol class="bibliography"><li><!-- _layouts/bib.html -->



      <div class="row">
<!--        <div class="col-sm-2 abbr"></div>
-->
        <!-- Entry bib key -->
        <div id="GRL2_Langenberger_David2009" class="col-sm-12" display="block">
        <span id="GRL2_Langenberger_David2009"><b><span style="font-style: bold">‚ÄúEvidence for human microRNA-offset RNAs in small RNA sequencing data‚Äù</span></b>. <br />
        D. Langenberger, C. Bermudez-Santana, J. Hertel, S. Hoffmann, P. Khaitovich, and P. F. Stadler. <br />
        <i>Oxford University Press</i>, 2009</span>
    
          <br />
          

          

           
            <a data-toggle="collapse" href="#GRL2_Langenberger_David2009-abstract" class="btn-abstract" role="button">ABS</a>
          

          <a href="http://dx.doi.org/10.1093/bioinformatics/btp419" target="_blank"><button class="btn-doi">DOI</button></a> 

          

          
          


           
            <a data-toggle="collapse" href="#GRL2_Langenberger_David2009-bib" class="btn-bib" role="button">BIB</a>
          


          

          
          <div class="collapse" id="GRL2_Langenberger_David2009-abstract">
            <div class="well-abstract">
            MicroRNA-offset-RNAs (moRNAs) were recently detected as highly abundant class of small RNAs in a basal chordate. Using short read sequencing data, we show here that moRNAs are also produced from human microRNA precursors, albeit at quite low expression levels. The expression levels of moRNAs are unrelated to those of the associated microRNAs. Surprisingly, microRNA precursors that also show moRNAs are typically evolutionarily old, comprising more than half of the microRNA families that were present in early Bilateria, while evidence for moRNAs was found only for a relative small fraction of microRNA families of recent origin.
            </div>
          </div>
          

           
          <div class="collapse" id="GRL2_Langenberger_David2009-bib">
            <div class="well-bib">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">GRL2_Langenberger_David2009</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Evidence for human microRNA-offset RNAs in small RNA sequencing data}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Oxford University Press}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Langenberger, David and Bermudez-Santana, Clara and Hertel, Jana and Hoffmann, Steve and Khaitovich, Philipp and Stadler, Peter F.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2009}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/bioinformatics/btp419}</span><span class="p">,</span>
  <span class="na">image_filename</span> <span class="p">=</span> <span class="s">{./images/publications/GRL2_Langenberger_David2009.png}</span><span class="p">,</span>
  <span class="na">published</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">tag</span> <span class="p">=</span> <span class="s">{10002}</span>
<span class="p">}</span></code></pre></figure>
          </div>
          </div>
          

       

          


        </div>

      </div></li></ol>

          </div>
        </div>
      </div>
    </div>
  </div>

  <div class="col-sm-6 clearfix">
    <div class="well">

      <p><img src="https://mchaib.github.io/BMTE-Website/images/publications/GenomeArchitectureAmanita.jpg" class="img-responsive" width="50%" style="float: top" /></p>

      <h3><pubtit>Rapid Divergence of Genome Architectures Following the Origin of an Ectomycorrhizal Symbiosis in the Genus *Amanita*</pubtit></h3>
      <p>Fungi are evolutionary shape shifters and adapt quickly to new environments. Ectomycorrhizal (EM) symbioses are mutualistic associations between fungi and plants and have evolved repeatedly and independently across the fungal tree of life, suggesting lineages frequently reconfigure genome content to take advantage of open ecological niches. To date analyses of genomic mechanisms facilitating EM symbioses have involved comparisons of distantly related species, but here, we use the genomes of three EM and two asymbiotic (AS) fungi from the genus Amanita as well as an AS outgroup to study genome evolution following a single origin of symbiosis. Our aim was to identify the defining features of EM genomes, but our analyses suggest no clear differentiation of genome size, gene repertoire size, or transposable element content between EM and AS species.</p>
      <p>Team members: <em>Jaqueline Hess, Inger Skrede, Maryam Chaib De Mares, Matthieu Hainaut, Bernard Henrissat, Anne Pringle</em></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
      <p><a data-toggle="collapse" href="#amanita-bib" class="btn-bib" style="text-decoration:none; color:#ebebeb; hover:#ebebeb;" role="button" aria-expanded="false">Selected papers</a></p>
      <div class="collapse" id="amanita-bib">
        <div class="well-abs">
          <div class="publications">
            <ol class="bibliography"><li><!-- _layouts/bib.html -->



      <div class="row">
<!--        <div class="col-sm-2 abbr"></div>
-->
        <!-- Entry bib key -->
        <div id="GRL2_Langenberger_David2009" class="col-sm-12" display="block">
        <span id="GRL2_Langenberger_David2009"><b><span style="font-style: bold">‚ÄúEvidence for human microRNA-offset RNAs in small RNA sequencing data‚Äù</span></b>. <br />
        D. Langenberger, C. Bermudez-Santana, J. Hertel, S. Hoffmann, P. Khaitovich, and P. F. Stadler. <br />
        <i>Oxford University Press</i>, 2009</span>
    
          <br />
          

          

           
            <a data-toggle="collapse" href="#GRL2_Langenberger_David2009-abstract" class="btn-abstract" role="button">ABS</a>
          

          <a href="http://dx.doi.org/10.1093/bioinformatics/btp419" target="_blank"><button class="btn-doi">DOI</button></a> 

          

          
          


           
            <a data-toggle="collapse" href="#GRL2_Langenberger_David2009-bib" class="btn-bib" role="button">BIB</a>
          


          

          
          <div class="collapse" id="GRL2_Langenberger_David2009-abstract">
            <div class="well-abstract">
            MicroRNA-offset-RNAs (moRNAs) were recently detected as highly abundant class of small RNAs in a basal chordate. Using short read sequencing data, we show here that moRNAs are also produced from human microRNA precursors, albeit at quite low expression levels. The expression levels of moRNAs are unrelated to those of the associated microRNAs. Surprisingly, microRNA precursors that also show moRNAs are typically evolutionarily old, comprising more than half of the microRNA families that were present in early Bilateria, while evidence for moRNAs was found only for a relative small fraction of microRNA families of recent origin.
            </div>
          </div>
          

           
          <div class="collapse" id="GRL2_Langenberger_David2009-bib">
            <div class="well-bib">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">GRL2_Langenberger_David2009</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Evidence for human microRNA-offset RNAs in small RNA sequencing data}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Oxford University Press}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Langenberger, David and Bermudez-Santana, Clara and Hertel, Jana and Hoffmann, Steve and Khaitovich, Philipp and Stadler, Peter F.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2009}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/bioinformatics/btp419}</span><span class="p">,</span>
  <span class="na">image_filename</span> <span class="p">=</span> <span class="s">{./images/publications/GRL2_Langenberger_David2009.png}</span><span class="p">,</span>
  <span class="na">published</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">tag</span> <span class="p">=</span> <span class="s">{10002}</span>
<span class="p">}</span></code></pre></figure>
          </div>
          </div>
          

       

          


        </div>

      </div></li></ol>

          </div>
        </div>
      </div>
    </div>
  </div>

</div>

<p> ¬† </p>

</div>

      </div>
    </div>

    <div id="footer" class="panel">
  <div class="panel-footer">
	<div class="container-fluid">
	  <div class="row">
		<div class="col-sm-4">




 <p>&copy 2025  BMTE Research Group.</p>
              <p>This is an initiative to publish the advancements of the BMTE (Theoretical and Evolutionary Molecular Biology)<a href="http://www.hermes.unal.edu.co/">Research group</a> at <a href="https://ciencias.bogota.unal.edu.co/estudiar_en_la_facultad/pregrados/biologia/">Universidad Nacional sede Bogot√°</a>.</p>
		   <p>  </p><p>
		</div>
		<div class="col-sm-4">
		  Main sponsor:<br />
		  - <a href="https://unal.edu.co/">Universidad Nacional sede Bogot√°</a> <br />
		</div>
		<div class="col-sm-4">
		  Contact:<br />
		  Universidad Nacional<br />
		  Carrera 45 N¬∞ 26-85 - Edificio Uriel Guti√©rrez<br />
		  Bogot√°<br />
		  Colombia<br />
            (<a href="https://maps.app.goo.gl/6fwE3pHMoeuEHbPZ7">Maps</a>, <a href="https://bogota.unal.edu.co/">UNAL sede Bogot√°</a>)




	</div>
	  </div>
	</div>
  </div>
</div>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script src="https://mchaib.github.io/BMTE-Website/js/bootstrap.min.js"></script>


  </body>

</html>
