<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>BMTE - Publications</title>
  <meta name="description" content="BMTE research Group -- Publications.">
  <link rel="stylesheet" href="https://mchaib.github.io/BMTE-Website/css/main.css">
  <link rel="canonical" href="https://mchaib.github.io/BMTE-Website/es/publications/">
<link rel="shortcut icon" type ="image/x-icon" href="https://mchaib.github.io/BMTE-Website/images/logopic/favicon.ico">
<link rel="shortcut icon" href="https://mchaib.github.io/favicon.png">


<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-26505344-1', 'auto');
  ga('send', 'pageview');

</script>


</head>


  <body>

    <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container-fluid">
	<div class="navbar-header">
	  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-1" aria-expanded="false">
		<span class="sr-only">Toggle navigation</span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span> 
	  </button>
		

    <a class="navbar-brand" href="https://mchaib.github.io/BMTE-Website/">
    	<img style="position:relative; top:-25px" src="/BMTE-Website/images/logopic/group_logo_medium.png" height="40" width="auto" class="d-inline-block align-left" alt=""> 
    <a class="navbar-brand" href="https://mchaib.github.io/BMTE-Website/">
			  MBTE Research Group
    </a>
  	</a>
	</div>

<div class="collapse navbar-collapse" id="navbar-collapse-1">
	  <ul class="nav navbar-nav navbar-right">
		<li><a href="https://mchaib.github.io/BMTE-Website/es/">Principal</a></li>
		<li><a href="https://mchaib.github.io/BMTE-Website/es/team">Integrantes</a></li>
		<li><a href="https://mchaib.github.io/BMTE-Website/es/allprojects">Proyectos en curso</a></li>
		<li><a href="https://mchaib.github.io/BMTE-Website/es/publications">Publicaciones</a></li>
		<li><a href="https://mchaib.github.io/BMTE-Website/es/synbio">SynBio</a></li>
	  </ul>
</div>

</div>

 <div class="language-switcher">
  
  <a href="https://mchaib.github.io/BMTE-Website//publications/">üá¨üáß / Eng</a>
  
</div>


</div>


    <div class="container-fluid">
      <div class="row">
        <div id="gridid" class="col-sm-12">
  <h1 id="publicaciones">Publicaciones</h1>
<h3 id="publicaciones-destacadas">Publicaciones destacadas</h3>
<p>(Para ver la lista completa de publicaciones, da click <a href="#lista-de-publicaciones">aqui</a></p>

<div class="row">

  <div class="col-sm-12 clearfix">
    <div class="well">
      <pubtit>Comparative Metagenomic Analysis of Biosynthetic Diversity across Sponge Microbiomes Highlights Metabolic Novelty, Conservation, and Diversification</pubtit>
      <p><img src="https://mchaib.github.io/BMTE-Website/images/publications/msystems.webp" class="img-responsive" width="40%" style="float: left" /></p>
      <p>Marine sponges and their microbial symbiotic communities are rich sources of diverse natural products (NPs) that often display biological activity, yet little is known about the global distribution of NPs and the symbionts that produce them. Since the majority of sponge symbionts remain uncultured, it is a challenge to characterize their NP biosynthetic pathways, assess their prevalence within the holobiont, and measure the diversity of NP biosynthetic gene clusters (BGCs) across sponge taxa and environments. Here, we explore the microbial biosynthetic landscapes of three high-microbial-abundance (HMA) sponges from the Atlantic Ocean and the Mediterranean Sea. This data set reveals striking novelty, with &lt;1% of the recovered gene cluster families (GCFs) showing similarity to any characterized BGC. When zooming in on the microbial communities of each sponge, we observed higher variability of specialized metabolic and taxonomic profiles between sponge species than within species. Nonetheless, we identified conservation of GCFs, with 20% of sponge GCFs being shared between at least two sponge species and a GCF core comprised of 6% of GCFs shared across all species. Within this functional core, we identified a set of widespread and diverse GCFs encoding nonribosomal peptide synthetases that are potentially involved in the production of diversified ether lipids, as well as GCFs putatively encoding the production of highly modified proteusins. The present work contributes to the small, yet growing body of data characterizing NP landscapes of marine sponge symbionts and to the cryptic biosynthetic potential contained in this environmental niche.</p>
      <p><em>Catarina Loureiro, Anastasia Galani, Asimenia Gavriilidou, etc.</em></p>
      <p><strong><a href="https://journals.asm.org/doi/10.1128/msystems.00357-22">Research Article, Facultad de ciencias de la Universidad Nacional Sede Bogot√°, (2022)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
    </div>
  </div>

  <div class="col-sm-12 clearfix">
    <div class="well">
      <pubtit>Rapid Divergence of Genome Architectures Following the Origin of an Ectomycorrhizal Symbiosis in the Genus *Amanita*</pubtit>
      <p><img src="https://mchaib.github.io/BMTE-Website/images/publications/GenomeArchitectureAmanita.jpg" class="img-responsive" width="40%" style="float: left" /></p>
      <p>Fungi are evolutionary shape shifters and adapt quickly to new environments. Ectomycorrhizal (EM) symbioses are mutualistic associations between fungi and plants and have evolved repeatedly and independently across the fungal tree of life, suggesting lineages frequently reconfigure genome content to take advantage of open ecological niches. To date analyses of genomic mechanisms facilitating EM symbioses have involved comparisons of distantly related species, but here, we use the genomes of three EM and two asymbiotic (AS) fungi from the genus Amanita as well as an AS outgroup to study genome evolution following a single origin of symbiosis. Our aim was to identify the defining features of EM genomes, but our analyses suggest no clear differentiation of genome size, gene repertoire size, or transposable element content between EM and AS species.</p>
      <p><em>Jaqueline Hess, Inger Skrede, Maryam Chaib De Mares, Matthieu Hainaut, Bernard Henrissat, Anne Pringle</em></p>
      <p><strong><a href="https://academic.oup.com/mbe/article/35/11/2786/5100885">Research Article, Facultad de ciencias de la Universidad Nacional Sede Bogot√°, (2022)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
    </div>
  </div>

</div>

<p> ¬† </p>

<h2 id="lista-de-publicaciones">Lista de publicaciones</h2>

<h3 id="grupo-de-investigaci√≥n-bmte">Grupo de Investigaci√≥n (BMTE)</h3>
<div class="publications">
  <ol class="bibliography"><li><!-- _layouts/bib.html -->



      <div class="row">
<!--        <div class="col-sm-2 abbr"></div>
-->
        <!-- Entry bib key -->
        <div id="GRL5_Mares_Chaib_2025" class="col-sm-12" display="block">
        <span id="GRL5_Mares_Chaib_2025"><b><span style="font-style: bold">‚ÄúDistinct bacteria display genus and species-specific associations with mycobionts in paramo lichens in Colombia‚Äù</span></b>. <br />
        M. C. D. Mares <i>et al.</i> <br />
        <i>Oxford Academic</i>, 2025</span>
    
          <br />
          

          

           
            <a data-toggle="collapse" href="#GRL5_Mares_Chaib_2025-abstract" class="btn-abstract" role="button">ABS</a>
          

          <a href="http://dx.doi.org/10.1093/femsec/fiaf010" target="_blank"><button class="btn-doi">DOI</button></a> 

          

          
          


           
            <a data-toggle="collapse" href="#GRL5_Mares_Chaib_2025-bib" class="btn-bib" role="button">BIB</a>
          


          

          
          <div class="collapse" id="GRL5_Mares_Chaib_2025-abstract">
            <div class="well-abstract">
            Lichens are complex symbiotic systems where fungi interact with an extracellular arrangement of one or more photosynthetic partners and an indeterminate number of other microbes. Recently, specific lichen‚Äìmicrobial community associations have been proposed. In this study, we aimed to characterize the differences in bacteria associated with closely related lichens, under a defined set of environmental conditions in Colombian paramos. Our goal was to determine if there is a correlation between microbiota and host divergence in lichen species belonging to the genus Sticta. We found that specific microbiota are defined by their mycobiont at the genus level. Further, distinct bacterial families show differences among the three studied genera, and specific amplicon sequence variants further discriminate among lichen species within each genus. A geographic component also determines the composition of these microbial communities among lichen species. Our functional analysis revealed that fungal partners play a key role in synthesizing complex polysaccharides, while bacterial-derived antioxidants and photoprotective mechanisms contribute to desiccation tolerance in lichens. These insights highlight the complex interactions within lichen symbioses that could be relevant in environments such as the paramo ecosystem.
            </div>
          </div>
          

           
          <div class="collapse" id="GRL5_Mares_Chaib_2025-bib">
            <div class="well-bib">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">GRL5_Mares_Chaib_2025</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Distinct bacteria display genus and species-specific associations with mycobionts in paramo lichens in Colombia}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Oxford Academic}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Mares, Maryam Chaib De and Castro, Emerson Arciniegas and Ulloa, Maria Alejandra and Torres, Jean Marc and Sierra, Maria A and Butler, Daniel J and Mason, Christopher E and Zambrano, Mar√≠a Mercedes and Moncada, Bibiana and Mu√±oz, Alejandro Reyes}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/femsec/fiaf010}</span><span class="p">,</span>
  <span class="na">image_filename</span> <span class="p">=</span> <span class="s">{}</span><span class="p">,</span>
  <span class="na">published</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">tag</span> <span class="p">=</span> <span class="s">{10001}</span>
<span class="p">}</span></code></pre></figure>
          </div>
          </div>
          

       

          


        </div>

      </div></li>
<li><!-- _layouts/bib.html -->



      <div class="row">
<!--        <div class="col-sm-2 abbr"></div>
-->
        <!-- Entry bib key -->
        <div id="GRL1_Mares_Chaib2018" class="col-sm-12" display="block">
        <span id="GRL1_Mares_Chaib2018"><b><span style="font-style: bold">‚ÄúExpressed protein profile of a Tectomicrobium and other microbial symbionts in the marine sponge Aplysina aerophoba as evidenced by metaproteomics‚Äù</span></b>. <br />
        M. C. D. Mares <i>et al.</i> <br />
        <i>Nature</i>, 2018</span>
    
          <br />
          

          

           
            <a data-toggle="collapse" href="#GRL1_Mares_Chaib2018-abstract" class="btn-abstract" role="button">ABS</a>
          

          <a href="http://dx.doi.org/10.1038/s41598-018-30134-0" target="_blank"><button class="btn-doi">DOI</button></a> 

          

          
          


           
            <a data-toggle="collapse" href="#GRL1_Mares_Chaib2018-bib" class="btn-bib" role="button">BIB</a>
          


          

          
          <div class="collapse" id="GRL1_Mares_Chaib2018-abstract">
            <div class="well-abstract">
            Aplysina aerophoba is an emerging model marine sponge, with a well-characterized microbial community in terms of diversity and structure. However, little is known about the expressed functional capabilities of its associated microbes. Here, we present the first metaproteomics-based study of the microbiome of A. aerophoba. We found that transport and degradation of halogenated and chloroaromatic compounds are common active processes in the sponge microbiomes
            </div>
          </div>
          

           
          <div class="collapse" id="GRL1_Mares_Chaib2018-bib">
            <div class="well-bib">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">GRL1_Mares_Chaib2018</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Expressed protein profile of a Tectomicrobium and other microbial symbionts in the marine sponge Aplysina aerophoba as evidenced by metaproteomics}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Mares, Maryam Chaib De and Jim√©nez, Diego Javier and Palladino, Giorgia and Gutleben, Johanna and Lebrun, Laura A. and Muller, Emilie E. L. and Wilmes, Paul and Sipkema, Detmer and van Elsas, Jan Dirk}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41598-018-30134-0}</span><span class="p">,</span>
  <span class="na">image_filename</span> <span class="p">=</span> <span class="s">{./images/publications/GRL1_Mares_Chaib2018.webp}</span><span class="p">,</span>
  <span class="na">published</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">tag</span> <span class="p">=</span> <span class="s">{10001}</span>
<span class="p">}</span></code></pre></figure>
          </div>
          </div>
          

       

          


        </div>

      </div></li>
<li><!-- _layouts/bib.html -->



      <div class="row">
<!--        <div class="col-sm-2 abbr"></div>
-->
        <!-- Entry bib key -->
        <div id="GRL2_Langenberger_David2009" class="col-sm-12" display="block">
        <span id="GRL2_Langenberger_David2009"><b><span style="font-style: bold">‚ÄúEvidence for human microRNA-offset RNAs in small RNA sequencing data‚Äù</span></b>. <br />
        D. Langenberger, C. Bermudez-Santana, J. Hertel, S. Hoffmann, P. Khaitovich, and P. F. Stadler. <br />
        <i>Oxford University Press</i>, 2009</span>
    
          <br />
          

          

           
            <a data-toggle="collapse" href="#GRL2_Langenberger_David2009-abstract" class="btn-abstract" role="button">ABS</a>
          

          <a href="http://dx.doi.org/10.1093/bioinformatics/btp419" target="_blank"><button class="btn-doi">DOI</button></a> 

          

          
          


           
            <a data-toggle="collapse" href="#GRL2_Langenberger_David2009-bib" class="btn-bib" role="button">BIB</a>
          


          

          
          <div class="collapse" id="GRL2_Langenberger_David2009-abstract">
            <div class="well-abstract">
            MicroRNA-offset-RNAs (moRNAs) were recently detected as highly abundant class of small RNAs in a basal chordate. Using short read sequencing data, we show here that moRNAs are also produced from human microRNA precursors, albeit at quite low expression levels. The expression levels of moRNAs are unrelated to those of the associated microRNAs. Surprisingly, microRNA precursors that also show moRNAs are typically evolutionarily old, comprising more than half of the microRNA families that were present in early Bilateria, while evidence for moRNAs was found only for a relative small fraction of microRNA families of recent origin.
            </div>
          </div>
          

           
          <div class="collapse" id="GRL2_Langenberger_David2009-bib">
            <div class="well-bib">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">GRL2_Langenberger_David2009</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Evidence for human microRNA-offset RNAs in small RNA sequencing data}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Oxford University Press}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Langenberger, David and Bermudez-Santana, Clara and Hertel, Jana and Hoffmann, Steve and Khaitovich, Philipp and Stadler, Peter F.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2009}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/bioinformatics/btp419}</span><span class="p">,</span>
  <span class="na">image_filename</span> <span class="p">=</span> <span class="s">{./images/publications/GRL2_Langenberger_David2009.png}</span><span class="p">,</span>
  <span class="na">published</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">tag</span> <span class="p">=</span> <span class="s">{10002}</span>
<span class="p">}</span></code></pre></figure>
          </div>
          </div>
          

       

          


        </div>

      </div></li>
<li><!-- _layouts/bib.html -->



      <div class="row">
<!--        <div class="col-sm-2 abbr"></div>
-->
        <!-- Entry bib key -->
        <div id="GRL4_Mares_Chaib2023" class="col-sm-12" display="block">
        <span id="GRL4_Mares_Chaib2023"><b><span style="font-style: bold">‚ÄúDesign of gene circuits in microorganisms to modulate host physiology‚Äù</span></b>. <br />
        E. B. Gomez, J. C. Naranjo, L. M. R. Loaiza, C. Vega, J. C. R. Medina, and F. Gomez. <br />
        <i>Sociedad Colombiana de Ciencias Qu√≠micas</i>, 2023</span>
    
          <br />
          

          

           
            <a data-toggle="collapse" href="#GRL4_Mares_Chaib2023-abstract" class="btn-abstract" role="button">ABS</a>
          

          

          

          
          


           
            <a data-toggle="collapse" href="#GRL4_Mares_Chaib2023-bib" class="btn-bib" role="button">BIB</a>
          


          

          
          <div class="collapse" id="GRL4_Mares_Chaib2023-abstract">
            <div class="well-abstract">
            This paper recapitulates a brief history of the emergence and definition of synthetic biology, summarizes the basic concepts to understand how it is practiced, describes how it articulates with industry in terms of innovation, and introduces the most recent applications of artificial intelligence in this area.
            </div>
          </div>
          

           
          <div class="collapse" id="GRL4_Mares_Chaib2023-bib">
            <div class="well-bib">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">GRL4_Mares_Chaib2023</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Design of gene circuits in microorganisms to modulate host physiology}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Sociedad Colombiana de Ciencias Qu√≠micas}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Gomez, Edgar Babativa and Naranjo, Julieth Camila and Loaiza, Luis Miguel Riveros and Vega, Catalina and Medina, Juan Carlos Riveros and Gomez, Francisco}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">image_filename</span> <span class="p">=</span> <span class="s">{./images/publications/GRL4_Mares_Chaib2023.png}</span><span class="p">,</span>
  <span class="na">published</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">tag</span> <span class="p">=</span> <span class="s">{10003}</span>
<span class="p">}</span></code></pre></figure>
          </div>
          </div>
          

       

          


        </div>

      </div></li>
<li><!-- _layouts/bib.html -->



      <div class="row">
<!--        <div class="col-sm-2 abbr"></div>
-->
        <!-- Entry bib key -->
        <div id="GRL3_Mares_Chaib2014" class="col-sm-12" display="block">
        <span id="GRL3_Mares_Chaib2014"><b><span style="font-style: bold">‚ÄúHorizontal transfer of carbohydrate metabolism genes into ectomycorrhizal Amanita‚Äù</span></b>. <br />
        M. C. D. Mares <i>et al.</i> <br />
        <i>New Phytologist Foundation</i>, 2014</span>
    
          <br />
          

          

           
            <a data-toggle="collapse" href="#GRL3_Mares_Chaib2014-abstract" class="btn-abstract" role="button">ABS</a>
          

          <a href="http://dx.doi.org/10.1111/nph.13140" target="_blank"><button class="btn-doi">DOI</button></a> 

          

          
          


           
            <a data-toggle="collapse" href="#GRL3_Mares_Chaib2014-bib" class="btn-bib" role="button">BIB</a>
          


          

          
          <div class="collapse" id="GRL3_Mares_Chaib2014-abstract">
            <div class="well-abstract">
            The genus Amanita encompasses both symbiotic, ectomycorrhizal fungi and asymbiotic litter decomposers; all species are derived from asymbiotic ancestors. Symbiotic species are no longer able to degrade plant cell walls. The carbohydrate esterases family 1 (CE1s) is a diverse group of enzymes involved in carbon metabolism, including decomposition and carbon storage. CE1 genes of the ectomycorrhizal A. muscaria appear diverged from all other fungal homologues, and more similar to CE1s of bacteria, suggesting a horizontal gene transfer (HGT) event.
            </div>
          </div>
          

           
          <div class="collapse" id="GRL3_Mares_Chaib2014-bib">
            <div class="well-bib">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">GRL3_Mares_Chaib2014</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Horizontal transfer of carbohydrate metabolism genes into ectomycorrhizal Amanita}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{New Phytologist Foundation}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Mares, Maryam Chaib De and Hess, Jaqueline and Floudas, Dimitrios and Lipzen, Anna and Choi, Cindy and Kennedy, Megan and Grigoriev, Igor V. and Pringle, Anne}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2014}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1111/nph.13140}</span><span class="p">,</span>
  <span class="na">link</span> <span class="p">=</span> <span class="s">{https://nph.onlinelibrary.wiley.com/doi/10.1111/nph.13140}</span><span class="p">,</span>
  <span class="na">image_filename</span> <span class="p">=</span> <span class="s">{./images/publications/GRL3_Mares_Chaib2014.webp}</span><span class="p">,</span>
  <span class="na">published</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">tag</span> <span class="p">=</span> <span class="s">{10004}</span>
<span class="p">}</span></code></pre></figure>
          </div>
          </div>
          

       

          


        </div>

      </div></li>
<li><!-- _layouts/bib.html -->



      <div class="row">
<!--        <div class="col-sm-2 abbr"></div>
-->
        <!-- Entry bib key -->
        <div id="GRL6_Mares_Chaib2018" class="col-sm-12" display="block">
        <span id="GRL6_Mares_Chaib2018"><b><span style="font-style: bold">‚ÄúRapid Divergence of Genome Architectures Following the Origin of an Ectomycorrhizal Symbiosis in the Genus Amanita‚Äù</span></b>. <br />
        J. Hess, I. Skrede, M. C. D. Mares, M. Hainaut, B. Henrissat, and A. Pringle. <br />
        <i>Oxford Academic</i>, 2018</span>
    
          <br />
          

          

           
            <a data-toggle="collapse" href="#GRL6_Mares_Chaib2018-abstract" class="btn-abstract" role="button">ABS</a>
          

          <a href="http://dx.doi.org/10.1093/molbev/msy179" target="_blank"><button class="btn-doi">DOI</button></a> 

          

          
          


           
            <a data-toggle="collapse" href="#GRL6_Mares_Chaib2018-bib" class="btn-bib" role="button">BIB</a>
          


          

          
          <div class="collapse" id="GRL6_Mares_Chaib2018-abstract">
            <div class="well-abstract">
            Fungi are evolutionary shape shifters and adapt quickly to new environments. Ectomycorrhizal (EM) symbioses are mutualistic associations between fungi and plants and have evolved repeatedly and independently across the fungal tree of life, suggesting lineages frequently reconfigure genome content to take advantage of open ecological niches. To date analyses of genomic mechanisms facilitating EM symbioses have involved comparisons of distantly related species, but here, we use the genomes of three EM and two asymbiotic (AS) fungi from the genus Amanita as well as an AS outgroup to study genome evolution following a single origin of symbiosis. Our aim was to identify the defining features of EM genomes, but our analyses suggest no clear differentiation of genome size, gene repertoire size, or transposable element content between EM and AS species. Phylogenetic inference of gene gains and losses suggests the transition to symbiosis was dominated by the loss of plant cell wall decomposition genes, a confirmation of previous findings. However, the same dynamic defines the AS species A. inopinata, suggesting loss is not strictly associated with origin of symbiosis. Gene expansions in the common ancestor of EM Amanita were modest, but lineage specific and large gene family expansions are found in two of the three EM extant species. Even closely related EM genomes appear to share few common features. The genetic toolkit required for symbiosis appears already encoded in the genomes of saprotrophic species, and this dynamic may explain the pervasive, recurrent evolution of ectomycorrhizal associations
            </div>
          </div>
          

           
          <div class="collapse" id="GRL6_Mares_Chaib2018-bib">
            <div class="well-bib">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">GRL6_Mares_Chaib2018</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Rapid Divergence of Genome Architectures Following the Origin of an Ectomycorrhizal Symbiosis in the Genus Amanita}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Oxford Academic}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Hess, Jaqueline and Skrede, Inger and Mares, Maryam Chaib De and Hainaut, Matthieu and Henrissat, Bernard and Pringle, Anne}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/molbev/msy179}</span><span class="p">,</span>
  <span class="na">link</span> <span class="p">=</span> <span class="s">{https://academic.oup.com/mbe/article/35/11/2786/5100885}</span><span class="p">,</span>
  <span class="na">image_filename</span> <span class="p">=</span> <span class="s">{}</span><span class="p">,</span>
  <span class="na">published</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">tag</span> <span class="p">=</span> <span class="s">{10004}</span>
<span class="p">}</span></code></pre></figure>
          </div>
          </div>
          

       

          


        </div>

      </div></li>
<li><!-- _layouts/bib.html -->



      <div class="row">
<!--        <div class="col-sm-2 abbr"></div>
-->
        <!-- Entry bib key -->
        <div id="GRL7_Mares_Chaib2022" class="col-sm-12" display="block">
        <span id="GRL7_Mares_Chaib2022"><b><span style="font-style: bold">‚ÄúFunctional and Phylogenetic Characterization of Bacteria in Bovine Rumen Using Fractionation of Ruminal Fluid‚Äù</span></b>. <br />
        R. Hern√°ndez, M. C. D. Mares, H. Jimenez, A. Reyes, and A. Caro-Quintero. <br />
        <i>frontiers</i>, 2022</span>
    
          <br />
          

          

           
            <a data-toggle="collapse" href="#GRL7_Mares_Chaib2022-abstract" class="btn-abstract" role="button">ABS</a>
          

          <a href="http://dx.doi.org/10.3389/fmicb.2022.813002" target="_blank"><button class="btn-doi">DOI</button></a> 

          

          
          


           
            <a data-toggle="collapse" href="#GRL7_Mares_Chaib2022-bib" class="btn-bib" role="button">BIB</a>
          


          

          
          <div class="collapse" id="GRL7_Mares_Chaib2022-abstract">
            <div class="well-abstract">
            Cattle productivity depends on our ability to fully understand and manipulate the fermentation process of plant material that occurs in the bovine rumen, which ultimately leads to the improvement of animal health and increased productivity with a reduction in environmental impact. An essential step in this direction is the phylogenetic and functional characterization of the microbial species composing the ruminal microbiota. To address this challenge, we separated a ruminal fluid sample by size and density using a sucrose density gradient. We used the full sample and the smallest fraction (5%), allowing the enrichment of bacteria, to assemble metagenome-assembled genomes (MAGs). We obtained a total of 16 bacterial genomes, 15 of these enriched in the smallest fraction of the gradient. According to the recently proposed Genome Taxonomy Database (GTDB) taxonomy, these MAGs belong to Bacteroidota, Firmicutes_A, Firmicutes, Proteobacteria, and Spirochaetota phyla
            </div>
          </div>
          

           
          <div class="collapse" id="GRL7_Mares_Chaib2022-bib">
            <div class="well-bib">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">GRL7_Mares_Chaib2022</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Functional and Phylogenetic Characterization of Bacteria in Bovine Rumen Using Fractionation of Ruminal Fluid}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{frontiers}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Hern√°ndez, Ruth and Mares, Maryam Chaib De and Jimenez, Hugo and Reyes, Alejandro and Caro-Quintero, Alejandro}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fmicb.2022.813002}</span><span class="p">,</span>
  <span class="na">link</span> <span class="p">=</span> <span class="s">{https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2022.813002/full}</span><span class="p">,</span>
  <span class="na">image_filename</span> <span class="p">=</span> <span class="s">{}</span><span class="p">,</span>
  <span class="na">published</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">tag</span> <span class="p">=</span> <span class="s">{10004}</span>
<span class="p">}</span></code></pre></figure>
          </div>
          </div>
          

       

          


        </div>

      </div></li></ol>

</div>

</div>

      </div>
    </div>

    <div id="footer" class="panel">
  <div class="panel-footer">
	<div class="container-fluid">
	  <div class="row">
		<div class="col-sm-4">




              <p>&copy 2025  BMTE Grupo de Investigaci√≥n.</p>
              <p>Esta es una iniciativa para publicar los avances del grupo de investigaci√≥n de Biolog√≠a Molecular Te√≥rica y Evolutiva <a href="http://www.hermes.unal.edu.co/">(BMTE)</a> en la <a href="https://ciencias.bogota.unal.edu.co/estudiar_en_la_facultad/pregrados/biologia/">Universidad Nacional sede Bogot√°</a>.</p>
                   <p>  </p><p>
                </div>
                <div class="col-sm-4">
                  Principal benefactor:<br />
                  - <a href="https://unal.edu.co/">Universidad Nacional sede Bogot√°</a> <br />
                </div>
                <div class="col-sm-4">
                  Lugar Principal:<br />
                  Universidad Nacional<br />
                  Carrera 45 N¬∞ 26-85 - Edificio Uriel Guti√©rrez<br />
                  Bogot√°<br />
                  Colombia<br />
            (<a href="https://maps.app.goo.gl/6fwE3pHMoeuEHbPZ7">Maps</a>, <a href="https://bogota.unal.edu.co/">UNAL sede Bogot√°</a>)






	</div>
	  </div>
	</div>
  </div>
</div>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script src="https://mchaib.github.io/BMTE-Website/js/bootstrap.min.js"></script>


  </body>

</html>
